guidance/_parser.py

```diff
@@ -654,9 +657,9 @@ def _compute_children(self, state_set_pos, item, reversed_state_sets, values_pos
             if self._compute_children(
                 state_set_pos, item, reversed_state_sets, values_pos + 1
             ):
-                item.children[values_pos] = (
-                    EarleyItem(value, tuple(), 0, state_set_pos, 0, state_set_pos)  # this child has zero length since it was nullable
-                )
+                item.children[values_pos] = EarleyItem(
+                    value, tuple(), 0, state_set_pos, 0, state_set_pos
+                )  # this child has zero length since it was nullable
```

Comment:
Small point, but shifting this comment to the line before (rather than trailing) will probably make the formatting easier.

Reply:
Ah, this was black overstepping a bit. Can make the edit!